### ISSUE TITLE
Customize replacement text for image in gallery element

### ISSUE THREAD

**Issue by rodolphebertozzo** [soflyy/breakdance-bugs#1055]
Currently, for each image in the gallery element, it is not possible to customize the replacement text as for the "image" element.
![image](https://github.com/soflyy/breakdance-bugs/assets/56270626/a10be1a2-a62b-4849-90da-cafcb0b230f8)
the alt text tab is missing

**Comment by rodolphebertozzo** [comment]
@louisreingold
